### DISCUSSION TITLE
quill mentioned people list

### DISCUSSION THREAD

**Discussion by hnegi2531** [afconsult/quill-mention#390] [Q&A] [answered]
Hey @MadSpindel @cgross @cwhite-connectfirst @eltimn 

how do i get the list of mentioned people

suppose i have list of people 
const list = ["john", "jack", "anna", "himanshu"]

and i mentioned @john and @jack so how do i get the list contains these two names
eg => mentioned = ["john", "jack"];

if i remove @jack from mentioned so my mentioned list should like this 
eg => mentioned = ["john"]

How can i achieve this kind of functionality

Thanks

**Comment by dmostafiz** [comment]
did you get the solution ?? i am really

**Comment by krishnaIVP** [comment, marked as answer]
with the help of this... you can achieve this:

```js
          mention: {
            allowedChars: /^[A-Za-z\sÅÄÖåäö]*$/,
            mentionDenotationChars: ['@'],
            onSelect: useCallback((item, insertItem) => {
              console.log(item)
              let selectedItem: IMentionItems = {
                id: item.id,
                value: item.value
              }
              let nm = item.value
              item.value = `{${nm}}`
              ref.current.editor.emitter.emit('mention-clicked', selectedItem)
              if (getMentionItem) getMentionItem(selectedItem)
              insertItem(item)
            }, []),
            source: renderMentionList
          }
```

My output for this is: @{ john }... you can customize this accordingly
